### PR TITLE
fix(storage.cold-archive): fix issue preventing display user-create info

### PR DESCRIPTION
**Pull request by brofik**
ref: MANAGER-9964

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-11016` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-9964 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/template.html [hunk 1]
```diff
@@ -131,13 +131,13 @@
         </oui-field>
 
         <!--User credential overview information-->
-        <pci-project-storages-user-credential-banner
+        <storages-components-user-credential-banner
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/template.html:134]:
```suggestion
        <pci-projects-project-storages-components-user-credential-banner
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/template.html (hunk 1 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/template.html:134]: Here, we are using the same component used for Cold-archive container creation (new user & linked user) and the same component for Object storage container creation (linked user).

The component name has already been validated in other review, changing its name is outside of the the scope of this PR.

Comment (anooparveti) [reply packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/template.html:134]: The PR which was validated before maybe a mistake. If you feel renaming the component is out of the scope of this PR, can you create a refactor task to handle the rename of the component?

**Review by mohammed-zahaf — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/template.html (hunk 1 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/template.html:134]: I've created the refactor task : MANAGER-11178

**Review by anooparveti — APPROVED** [review]

**Pull request merged**